http/logical.go

```diff
@@ -75,6 +75,30 @@ func buildLogicalRequest(core *vault.Core, w http.ResponseWriter, r *http.Reques
 		}
 	}
 
+	// If we are a read operation, try and parse any parameters
+	if op == logical.ReadOperation {
+		getData := map[string]interface{}{}
+
+		for k, v := range r.URL.Query() {
+			// Skip the help key as this is a reserved parameter
+			if k == "help" {
+				continue
+			}
+
+			switch {
```

Comment:
Should we scope this condition to apply only when k == "version"?

Reply:
No, we want to support any generic parameter

Reply:
Ahh yes, I misread this.